yarn.lock

```diff
@@ -3320,10 +3320,10 @@ mem@^1.1.0:
   dependencies:
     mimic-fn "^1.0.0"
 
-merge@1.2.0:
-  version "1.2.0"
-  resolved "https://registry.yarnpkg.com/merge/-/merge-1.2.0.tgz#7531e39d4949c281a66b8c5a6e0265e8b05894da"
-  integrity sha1-dTHjnUlJwoGma4xabgJl6LBYlNo=
+merge@1.2.1:
```

Comment:
Remove this change

Reply:
From here I guess: https://github.com/Shopify/eslint-plugin-shopify/commit/d45d8d07dfabe83e80d48661b235e72c0e123e9e